app/controls/details/details.js

```diff
@@ -115,6 +115,7 @@ define([
             }
 
             Booking.vouchers = this.options.vouchers || {};
+            Booking.webVouchers = this.options.booking.webVouchers || {};
```

Comment:
You shouldn't need to attach this at this point. It will just hang around not doing anything here. ```Booking``` is the constructor, ```this.options.booking``` is an instantiation of the object. If Drupal is returning the webVouchers object is /property/booking responses then it will be attached. Guarding against it being undefined by providing a default value should be enough.